gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/io/graphml/GraphMLWriter.java

```diff
@@ -58,13 +58,14 @@
  */
 public final class GraphMLWriter implements GraphWriter {
     private final XMLOutputFactory inputFactory = XMLOutputFactory.newInstance();
-    private boolean normalize = false;
+    private boolean normalize = true;
```

Comment:
I don't think we should normalize by default. We should offer the cheapest/fastest option as the default.